### DISCUSSION TITLE
Package reference

### DISCUSSION THREAD

**Discussion by alexachso** [Live-Charts/LiveCharts2#428] [General]
Hi,

I've installed the LiveChartsCore NuGet package in my project in order to use LiveCharts2. It naturally installed other NuGet packages that are referenced by LiveCharts2. 

My project is a simple WPF application for windows. When I compile the project it flags a reference warning:
![image](https://user-images.githubusercontent.com/12586000/161058675-12dde61c-04d2-4076-8814-91bab801ef76.png)

I manually traveled the path suggested and the file is there on the machine. I do not understand why Visual Studio says it cannot be found.

Can anyone give me a hint on what might be causing this warning?

**Comment by mayank07072019** [comment]
Hi Alexachso,

Did you find any solution ? I have the same issue.
